docs/4_configuration.rst

```diff
@@ -111,6 +111,8 @@ The following ``settings.py`` options are available for customizing Axes behavio
   Default: ``False``
 * ``AXES_ALLOWED_CORS_ORIGINS``: Configures lockout response CORS headers for XHR requests.
   Default: ``*``
+* ``AXES_SENSITIVE_PARAMS``: Configures POST and GET parameter values to mask in login attempt logging.
```

Comment:
Configuration key is `settings.AXES_SENSITIVE_PARAMETERS`, should this be the same?

Reply:
I've made the naming consistent